package.json

```diff
@@ -31,6 +31,8 @@
   },
   "homepage": "https://github.com/neomad-team/neomad.org#readme",
   "dependencies": {
+    "preact": "^8.1.0",
+    "preact-compat": "^3.16.0",
     "react": "^15.5.4",
```

Comment:
I still need because when I dev (yes yes I Dev ^^) I use react (super cool) tools in firefox. Preact don't have ...